staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -927,7 +927,8 @@ func (e *Store) updateForGracefulDeletionAndFinalizers(ctx context.Context, name
 			if !graceful {
 				// set the DeleteGracePeriods to 0 if the object has pendingFinalizers but not supporting graceful deletion
 				if pendingFinalizers {
-					klog.V(6).Infof("update the DeletionTimestamp to \"now\" and GracePeriodSeconds to 0 for object %s, because it has pending finalizers", name)
+					klog.V(6).InfoS("Object has pending finalizers, so the registry is going to update its status to deleting",
+						"object", name, "DeletionTimestamp", "\"now\"", "GracePeriodSeconds", "0")
```

Comment:
Let's always use `klog.KRef` when filling `object` key. Could you check in code if we could extract namespace from somewhere?

```suggestion
						"object", klog.KRef(namespace, name)
```